### PR TITLE
Fix broken e2e build job

### PR DESCRIPTION
**Pull request by htrqlik**
The problem is that `DocID=$DocID npm run test:e2e` is started before `npm run start:cov &` is ready. We still need to spawn a new process for `parcel` but we need to add additional logic to handle the *async* behaviour...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
   "scripts": {
     "build": "cross-env NODE_ENV=production parcel build src/index.html",
     "start": "parcel src/index.html --open",
-    "start:cov": "cross-env NODE_ENV=test parcel src/index.html",
+    "start:build": "cross-env NODE_ENV=test parcel build src/index.html",
+    "start:serve": "cross-env NODE_ENV=test parcel servce src/index.html",
```
Comment (axelssonHakan) [inline package.json]: TYPO servce

File: .circleci/config.yml [hunk 2]
```diff
@@ -43,7 +43,8 @@ jobs:
           name: E2E tests
           command: |
             DocID=$(curl -s --data-binary @data/drugcases.qvf http://localhost:9076/v1/apps/import | jq -r '.attributes.id')
-            npm run start:cov &
+            npm run start:build
+            npm run start:serve &
```
Comment (axelssonHakan) [inline .circleci/config.yml]: I think that serve will build it anyway :-(
But maybe faster

Comment (htrqlik) [reply .circleci/config.yml]: Something is still wrong. But I think I'm on the right track with this. It's still failing now with this approach. I'm looking at it...

Comment (htrqlik) [reply .circleci/config.yml]: I had to switch server to start using `http-server`. The build is now done synchronously and after that we start an http server using the dist folder.

**Comment by htrqlik** [comment]
Once this PR is merged we should be able to close https://github.com/qlik-oss/catwalk/pull/318

**Pull request merged**